src/main/webapp/web.xml

```diff
@@ -5,11 +5,55 @@
          version="3.1">
     <servlet>
         <servlet-name>index</servlet-name>
-        <servlet-class>mate.controller.IndexController</servlet-class>
+        <servlet-class>mate.controllers.IndexController</servlet-class>
     </servlet>
-    
     <servlet-mapping>
         <servlet-name>index</servlet-name>
-        <url-pattern>/index</url-pattern>
+        <url-pattern>/</url-pattern>
+    </servlet-mapping>
+
+    <servlet>
+        <servlet-name>createNewDriver</servlet-name>
+        <servlet-class>mate.controllers.CreateNewDriverController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>createNewDriver</servlet-name>
+        <url-pattern>/driver/create</url-pattern>
```

Comment:
https://mate-academy.github.io/jv-program-fulltime/02_jdbc/checklist/08_servlets_practice_checklist.html 
check urls naming for each url